rfc/offset-management.md

```diff
@@ -0,0 +1,224 @@
+## Scope
+
+Define a minimal offset management functionality for integration into fluvio.
+
+The initial implementation eg would default for a store of "memory", with a retention of 24 hours, and a count of 100 consumer groups of single consumers. Later expansion could allow for more durable storage (for example, internal partition), higher consumer group counts, multiple member consumer groups, and longer retention times.
```

Comment:
@digikata - I suggest we remove the default to `memory`, as the whole point of offset management is to make them durable. Also I don't think that a solution that does not recover from durable storage is viable.